regression/contracts/quantifiers-forall-both-01/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+main.c
+--enforce-all-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+The purpose of this test is to ensure that we can safely use __CPROVER_forall
+in a negative context. By using the --enforce-all-contracts flag, this test
```

Comment:
We cannot *always* safely use `__CPROVER_forall` in negative contexts, correct? I'd just add this here too. Same for `__CPROVER_exists` test cases.